app/models/key_pair.rb

```diff
@@ -3,6 +3,7 @@ class KeyPair < ActiveRecord::Base
   validates_lengths_from_database
   validates :name, :secret, :presence => true
   validates :compute_resource_id, :presence => true, :uniqueness => true
+  audited
```

Comment:
![image](https://cloud.githubusercontent.com/assets/433583/21131041/1bad139c-c115-11e6-875c-5e5ee4b4d1fc.png)

And

![image](https://cloud.githubusercontent.com/assets/433583/21131033/06501f8a-c115-11e6-9c19-f8e9e747ad69.png)
 
Seems ok, right?